delta_shooter/models.py

```diff
@@ -0,0 +1,268 @@
+from abc import ABC, abstractmethod
+from typing import Any, List, Literal, Tuple, Union
+
+import pygame
+from pygame.math import Vector2
+from pygame.surface import Surface
+from pygame.transform import rotozoom
+
+from shooter_utils import edge_barriers, get_random_velocity, load_sound, load_sprite, wrap_position
+
+UP = Vector2(0, -1)
+DOWN = Vector2(0, 1)
+RIGHT = Vector2(1, 0)
+LEFT = Vector2(-1, 0)
+
+
+GAME_SIZE = (600, 450)
+
+
+class DummyScreen:
+    def __init__(self, size: Tuple):
+        self.size = size
+        self.rect = pygame.Rect(0, 0, size[0], size[1])
+
+    def get_size(self) -> Tuple[int, int]:
+        return self.size[0], self.size[1]
```

Comment:
Why does this need to exist? Doesn't `self.size` work fine?

Reply:
it's to mock a method on a pygame Surface 

pygame.org/docs/ref/surface.html#pygame.Surface.get_size